joblib/format_stack.py

```diff
@@ -266,7 +261,7 @@ def linereader(file=file, lnum=[lnum], getline=linecache.getline):
         try:
             # This builds the names list in-place by capturing it from the
             # enclosing scope.
-            for token in generate_tokens(linereader):
+            for token in tokenize.generate_tokens(linereader):
```

Comment:
This looks reverse compared to what the comments says in l.32/34 but I have no clue what is the right behavior here as it seems like both exists in both version.

Reply:
Now that you point this out, it seems like this whole file can be removed anyway (!)